test/integration/webhook/operatingsystemconfig/operatingsystemconfig_test.go

```diff
@@ -145,15 +145,15 @@ var _ = Describe("Webhook tests", func() {
 					},
 				},
 			}
-			tmpl, err := template.New("config").Parse(string(testdata.GetSimpleRsyslogConfig()))
+			tmpl, err := template.New("config").Parse(string(webhooktest.GetSimpleRsyslogConfig()))
 			Expect(err).NotTo(HaveOccurred())
 			var simpleRsyslogConfig bytes.Buffer
 			Expect(tmpl.Execute(&simpleRsyslogConfig, configData)).To(Succeed())
 
-			expectedFiles = append(expectedFiles, testdata.GetRsyslogFiles(simpleRsyslogConfig.Bytes(), true)...)
-			expectedFiles = append(expectedFiles, testdata.GetAuditRulesFiles(true)...)
+			expectedFiles = append(expectedFiles, webhooktest.GetRsyslogFiles(simpleRsyslogConfig.Bytes(), true)...)
+			expectedFiles = append(expectedFiles, webhooktest.GetAuditRulesFiles(true)...)
 			Expect(osc.Spec.Files).To(HaveLen(len(expectedFiles)))
```

Comment:
The `ConsistOf` matcher should fail if there are missing or extra elements: ref https://github.com/onsi/gomega/blob/bae65a0fbd5d3a3ad901f70843fa651c8929cd83/matchers/consist_of.go#L157-L173

In my previous suggested, I was suggesting to drop also the expectation for the length of the files. It does not add value - `ConsistOf` already fails the lengths match but content - does not.

```suggestion
```